Makefile

```diff
@@ -308,23 +296,41 @@ verify-extended: check-generate check format test-cov test-cov-clean test-integr
 #####################################################################
 
 kind-% kind2-% gardener-%: export IPFAMILY := $(IPFAMILY)
+# KUBECONFIG
 kind-up kind-down gardener-up gardener-dev gardener-debug gardener-down register-local-env tear-down-local-env register-kind2-env tear-down-kind2-env test-e2e-local-simple test-e2e-local-migration test-e2e-local ci-e2e-kind ci-e2e-kind-upgrade: export KUBECONFIG = $(GARDENER_LOCAL_KUBECONFIG)
```

Comment:
The handling of `ci-e2e-kind-`  and `test-e2e-*` make targets is not consistent yet. 
For some of those targets their `KUBECONFIG` is listed here, for others in their respective targets themselves ([example](https://github.com/gardener/gardener/blob/2c7650d60e70f5289911e5c181ab1016050dc089/Makefile#L510)).
The lines for `KUBECONFIG` are already quite long. Maybe we could move those configs ti a similar section for `ci-*` and `test-*`.

Reply:
I am not sure I understand. From my perspective, it looks like the special handling for `ci-e2e-kind-migration-...` is required. Apart from that, I do not see any special `KUBECONFIG` handling.